libraries/content-tree-to-string/test.js

```diff
@@ -0,0 +1,53 @@
+import t from "tap"
```

Comment:
Perhaps we could get away with using node's built in test-runner / assert library for this?? https://nodejs.org/api/test.html